GAME/DELTAPAK.C

```diff
@@ -32,6 +32,7 @@
 #include <stddef.h>
 
 #include "DRAWPHAS.H"
+#include "SOUND.H"
```

Comment:
GAME/SPEC includes must be before standard library includes.

Reply:
Done